ROMFS/px4fmu_common/init.d/airframes/60002_uuv_bluerov2_heavy

```diff
@@ -23,14 +23,9 @@
 . ${R}etc/init.d/rc.uuv_defaults
 
 
-# disable circuit breaker for airspeed sensor
-param set-default CBRK_AIRSPD_CHK 162128
 # companion computer is connected via USB permanently
-param set-default CBRK_USB_CHK 197848
```

Comment:
Is this now off by default?

Reply:
Yes https://github.com/PX4/PX4-Autopilot/pull/17379/commits/aa0bffb73d17fe88b43aeaae151716a4a9fda7f3